endpoints/currency_rates.go

```diff
@@ -24,7 +25,7 @@ type rateConverter interface {
 }
 
 // newCurrencyRatesInfo creates a new CurrencyRatesInfo instance.
-func newCurrencyRatesInfo(rateConverter rateConverter) currencyRatesInfo {
+func newCurrencyRatesInfo(rateConverter rateConverter, rateConverterTickerTask *task.TickerTask) currencyRatesInfo {
```

Comment:
This doesn't need the `rateConverterTickerTask` though - right?, just the fetching interval. How about passing the fetching interval directly?